template-for-repository/negative_tests/README.md

```diff
@@ -0,0 +1,13 @@
+## Negative Tests
+
+This directory contains negative checks to ensure that CBMC CI jobs are run
+with the complete set of property-checking flags
+(see `CHECKFLAGS` in [Makefile.common](../proofs/Makefile.common))
+which we consider to be part of the best practice.
+
+To enable these tests in CI jobs,
+copy the negative test directories into `../proofs`.
```

Comment:
copy the directory negative_tests to proofs/negative_tests.  (We don't want them to pollute the proofs directory with two dozen directories, just one.)

Reply:
Good point! Thanks, lemme fix this.